### PR TITLE
[EmbeddingUsecase] add android tests full xml auto part

### PR DESCRIPTION
**Pull request by jacky-young**
add android tests full xml auto part

### CODE REVIEW

**Comment by jacky-young** [comment]
@Honry please help to review it.

File: usecase/usecase-embedding-android-tests/tests.full.xml [hunk 1]
```diff
@@ -1046,5 +1046,77 @@
         </description>
       </testcase>
     </set>
+    <set name="XWalkView-Auto" type="js" ui-auto="bdd">
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="PauseJsTimers" platform="android" priority="P0" purpose="Pause js timers Test With XWalkActivity" status="approved" type="functional_positive">
```
Comment (Honry) [inline usecase/usecase-embedding-android-tests/tests.full.xml]: You miss: status="approved" type="functional_positive"

File: usecase/usecase-embedding-android-tests/tests.auto.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/onIconAvailableOnReceivedIconAsync.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="Versions" platform="android" priority="P0" purpose="Versions Test With XWalkActivity" status="approved" type="functional_positive">
+        <description>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/usecase-embedding-android-tests/testscripts/Versions.feature</bdd_test_script_entry>
+        </description>
+      </testcase>
+      <testcase component="Crosswalk Use Cases/Embedding API" execution_type="auto" id="VersionsAsync" platform="android" priority="P0" purpose="Versions Test With XWalkInitializer" status="approved" type="functional_positive">
```
Comment (Honry) [inline usecase/usecase-embedding-android-tests/tests.auto.xml]: <status="approved" type="functional_positive" platform="android" priority="P0">
is no needed in tests.auto.xml, we should keep tests.auto.xml simplified.

File: usecase/usecase-embedding-android-tests/tests.full.xml (hunk 1 above)
Comment (jacky-young) [inline usecase/usecase-embedding-android-tests/tests.full.xml]: it has in the end, please check.

**Comment by Honry** [comment]
LGTM

**Pull request merged**